### PR TITLE
Add test for posegment

### PR DESCRIPTION
**Pull request by miurahr**
- add a missing test for posegment

Signed-off-by: Hiroshi Miura <miurahr@linux.com>

### CODE REVIEW

**Comment by unho** [comment]
Several lines are obviously longer than 80 characters. We enforce a 79 characters line length with a soft limit of 84 characters for some special cases. See our styleguide http://docs.translatehouse.org/projects/translate-toolkit/en/latest/developers/styleguide.html

File: translate/tools/test_posegment.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#: test/test.py:112
+msgid ""
+"Please let us know if you have any specific needs (A/V requirements, "
+"multiple microphones, a table, etc).  Note for example that 'audio out' is "
+"not provided for your computer unless you tell us in advance."
+msgstr ""
+"特に必要な物(A/V機器、複数のマイク、テーブルetc)があれば教えて下さい。例とし"
+"て、コンピュータからの「音声出力」は事前にお知らせ頂いていない場合は提供でき"
+"ないことに注意して下さい。"
+
```
Comment (unho) [inline translate/tools/test_posegment.py]: Is this blank line actually necessary?

**Comment by dwaynebailey** [comment]
@unho actually in tests we're pretty soft and have PEP8 turned off IIRC

File: translate/tools/test_posegment.py (hunk 1 above)
Comment (miurahr) [inline translate/tools/test_posegment.py]: This test case is come from live application.
A long line is copied from test_pogrep.
I'd like to update to drop blank line.

**Pull request closed without merging**